lib/buildkite/test_collector/http_client.rb

```diff
@@ -28,6 +28,28 @@ def post
       http.request(contact)
     end
 
+    def post_json(data)
+      contact_uri = URI.parse(url)
+
+      http = Net::HTTP.new(contact_uri.host, contact_uri.port)
+      http.use_ssl = contact_uri.scheme == "https"
```

Comment:
Is contact_uri just the buildkite URL? I see this is a pattern above, but why we don't just set this to true?

Reply:
The URL is configurable here:
https://github.com/buildkite/test-collector-ruby/blob/4f2ee8dae18274ecceb1aa3ffadedc1d60591ab4/lib/buildkite/test_collector.rb#L52

So for instance when we use the test collector in dev, the contact uri scheme is not https
https://github.com/buildkite/buildkite/blob/b34312e4b7d643eaeae064cc0b387ab8927872db/spec/spec_helper.rb#L8